### DISCUSSION TITLE
Support inline mode for Skeleton component

### DISCUSSION THREAD

**Discussion by mcmxcdev** [mantinedev/mantine#2284] [Feature requests]
The https://mantine.dev/core/skeleton/ component is quite simple for now and only allows elements to be placed on top of each other.

It would be great to support some kind of inline mode to be able to e.g. place a `Skeleton` with `circle` prop next to other placeholders.

**Comment by zhulien-ivanov** [comment]
Well, it does what it is designed for. It renders a simple `div` under the hood. Use `Group` or a simple `div` with `flexbox` to put them on the same line.

**Reply by mcmxcdev** [reply]
I have tried that before, and it ends up being squished and not looking as expected. I will have another look though.

**Comment by zhulien-ivanov** [comment]
Works fine for me: https://codesandbox.io/s/fast-breeze-3y5xn6?file=/src/App.js.

**Reply by mcmxcdev** [reply]
Indeed, thank you! Must've messed up somewhere.
